### PR TITLE
grammar edits and fixing k_infinity

### PR DESCRIPTION
**Pull request by mbmcgarry**
(trying again!) I did some grammatical editing, a few in-line comments where I was confused. One general comment: I'm not sure if it's 'neutronic poisoning' or 'neutron poisoning'
Otherwise looks good!

### CODE REVIEW

File: FCO.tex [hunk 1]
```diff
@@ -105,60 +105,59 @@ \section{Fix mixing ratio vs Pu-equivalent theory}
   \caption{Evolution of the plutonium content in the J1'' and  E3'' stream\label{fig:MW_flow} }
 \end{figure}
 
-The difference on J1'' and E3'' between all 6 calculations can be observed on
-Figure \ref{fig:MW_flow}. First one should only consider in this study the time
-after 15 and 75y, as the calculation need almost 12y to rich an equilibrium. 
+The difference between J1'' and E3'' for all 6 calculations can be observed in
+Figure \ref{fig:MW_flow}. Note that for this study one should only consider the time
+after 15 and 75y, as the calculation needs almost 12y to reach an equilibrium.
```
Comment (mbmcgarry) [inline FCO.tex:110]: Should this say "after 15 and 75yr" or "between 15 and 75yr"?

Comment (bam241) [inline FCO.tex:110]: I have should removed the 75yr mention (it is an old one... I believed I did... sorry)

File: FCO.tex [hunk 2]
```diff
@@ -222,20 +221,20 @@ \subsection{Decay}
...
-as a neutronics poisons), both of those effect tends to increase the amount of
-plutonium require to build the fuel.
-The effect of the presence of the initial inventory which is required to start
+degradation of the plutonium quality and the production of $^{241}$Am (acting
+as a neutronics poison), both of those effects tend to increase the amount of
+plutonium required to build the fuel.
+The effect of the presence of the initial inventory that is required to start
 the calculation, is as limited as possible: all calculation have been tuned to
-reduce at the minimum the amount of initial storage (which depends on the model
+minimize the amount of initial storage (which depends on the model
```
Comment (mbmcgarry) [inline FCO.tex:232]: This sentence "The effect on the presence. . . ." is unclear.

File: FCO.tex [hunk 3]
```diff
@@ -252,41 +251,40 @@ \subsection{Decay}
...
   \caption{Evolution of the plutonium composition in the J1'' storage in the
-  case of the initial inventory has been increased by a factor 10.}
+  case where the initial inventory has been increased by a factor 10.}
   \label{fig:LII_compo_x10}
 \end{figure}
 
 As shown figure \ref{fig:LII_compo_x10}, the fraction of $^{241}$Pu drops from
-about $12\%$ to less than $2\%$in about 60y, while the $^{241}$Am rises from
+about $12\%$ to less than $2\%$ in about 60y, while the $^{241}$Am rises from
 $5\%$ to about $12\%$. To illustrate this effect one only show the configuration
```
Comment (mbmcgarry) [inline FCO.tex:282]: This sentence "To illustrate this effect. . .. " is unclear

File: FCO.tex [hunk 4]
```diff
@@ -296,65 +294,62 @@ \section{Plutonium stacking effect}
...
+the MOX fuel fabrication, including whether or not decay is considered.
+The Pu-equivalence theory reaches an equilibrium (in the mixing ratio of E3'' and
+J1'' stream) very close to the fixed mixing ratio when the decay is not activated.
+This was expected: the Pu-equivalent model tried to build a MOX fuel with the same
+initial reactivity as the one used in the fixed mixing ratio calculation, so it
 used the same mixing ratio\dots  
 
-Nevertheless we can observe a variation of $20\%$ when taking into account the
-decay process in the contribution of E3'' and J1'' stream. This variation is a
+Nevertheless we can observe a variation of $20\%$ in the contribution of E3'' and J1'' streams  when taking the
```
Comment (mbmcgarry) [inline FCO.tex:316]: a variation of 20% in what? (I reordered the sentence based on my guess, but it may be wrong now)

File: FCO.tex [hunk 5]
```diff
@@ -296,65 +294,62 @@ \section{Plutonium stacking effect}
...
-cycle. Having very strong constrains might lead to ignore such disruption
-effect. In order to mimic a potential material stacking, the same steady state
-calculation have been perform three times increasing the amount of initial
-inventory require to start the calculation by a factor 2, 5 and 10. In all
-cases, $^{241}$Pu stacking induce $^{241}$Am production which impact strongly
-the mixing ration between E3'' and J1'' ( $5\%$ to $10\%$), which might have a
-strong effect on the used fuel composition, specially with thermal reactor, were
-the different cross section are so sensitive to the composition of the fuel.
+Moreover, the last part of the study shows the impact of the stacked plutonium
+on its composition and on fuel fabrication. This kind of material
```
Comment (mbmcgarry) [inline FCO.tex:344]: The sentence "This kind of material. .. ." is unclear

File: FCO.tex [hunk 6]
```diff
@@ -105,60 +105,59 @@ \section{Fix mixing ratio vs Pu-equivalent theory}
...
-to decrease the reactivity potential of the J1'' stream, which is compensate by
+is not affected much by the decay process as it contains mainly 239Pu, while the J1''
+stream contains a higher $^{241}$Pu content and is very sensitive to decay.  The decay tends
+to decrease the reactivity potential of the J1'' stream, which is compensated by
 the increase of the E3'' stream in the mix.
 
 
 \section{Model enrichment prediction}
 
 The second part of this memo is dedicated to the comparison of the plutonium
```
Comment (mbmcgarry) [inline FCO.tex:124]: This section talks about "kinds" of models and "categories" of models. I'm not sure if you're using the two words interchangeably to compare two things or if you're comparing two sets of things (4 total?)  If only 2, choose "kind" or "category" and be consistent. If you're comparing 4 combinations then give each one a name and then refer back to it explicitly

**Pull request merged**